packages/hardhat-ignition-core/test-integrations/new-api/internal/new-execution/jsonrpc-client.ts

```diff
@@ -121,6 +121,22 @@ describe("JSON-RPC client", function () {
           assert.equal(fees.maxPriorityFeePerGas, 1n);
         });
 
+        it("Should use the configured maxFeePerGas", async function () {
+          const maxFeeClient = new EIP1193JsonRpcClient(
+            this.hre.network.provider,
+            {
+              maxFeePerGas: 1n,
+              maxPriorityFeePerGas: 0n,
+            }
+          );
+          const fees = await maxFeeClient.getNetworkFees();
+
+          assert("maxFeePerGas" in fees);
+
+          // internal logic multiplies the given maxFeePerGas by 2 and adds the maxPriorityFeePerGas
+          assert.equal(fees.maxFeePerGas, 2n);
```

Comment:
Shouldn't this have been updated too? Tests seem to pass though, so I'm confused about the `2`.

Reply:
I think integration tests are disable right now on CI for some reason. This would also explain that one other test I mentioned that was failing for me locally despite CI passing.

Reply:
Good catch, I just pushed a fix for this test (though they still won't run on CI yet)